packages/tasks/src/model-libraries.ts

```diff
@@ -448,6 +448,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "mindspore",
 		repoUrl: "https://github.com/mindspore-ai/mindspore",
 	},
+	mitie:{
```

Comment:
Can you move this entry below between "mesh-anything" and "ml-agents" to preserve alphabetical order? Thanks! :pray:

Reply:
Np, will do!